src/consensus/aft/raft.h

```diff
@@ -483,6 +525,12 @@ namespace aft
       return get_latest_configuration_unsafe().size();
     }
 
+    ccf::SeqNo get_confirmed_matching_index(const NodeId& id) const
```

Comment:
We've almost entirely moved away from magic values and towards optionals, I think we should try to stick to that.